plugins/inputs/aerospike/aerospike.go

```diff
@@ -176,6 +232,29 @@ func copyTags(m map[string]string) map[string]string {
 	return out
 }
 
+func splitNamespaceAndOperation(hist string) (string, string) {
+	if strings.Contains(hist, "{") {
+		indexOfOpenCurlyBracket := strings.Index(hist, "{")
+		indexOfCloseCurlyBracket := strings.Index(hist, "}")
+		namespace := hist[indexOfOpenCurlyBracket+1 : indexOfCloseCurlyBracket]
```

Comment:
I think we are missing test cases what if there is no `}`?  What about `{}`?  We should be safe against any response that can come from Aerospike, not just valid responses.

Reply:
good point @danielnelson, I will add a check for all cases.